modules/src/ics07_tendermint/client_state.rs

```diff
@@ -115,10 +115,7 @@ impl ClientState {
     /// Resets all fields except the blockchain-specific ones.
     pub fn zero_custom_fields(mut client_state: Self) -> Self {
         client_state.trusting_period = ZERO_DURATION;
-        client_state.trust_level = TrustThresholdFraction {
-            numerator: 0,
-            denominator: 0,
-        };
+        client_state.trust_level = TrustThresholdFraction::default();
```

Comment:
@adizere I took this from https://github.com/informalsystems/ibc-rs/pull/1120/commits/0c797c8af36b764e8cdb5305b0e46178877dfdb4 but am not sure that this is right, since the `Default` instance for `TrustThresholdFraction` returns `ONE_THIRD`.

Reply:
Indeed, this might break the chain upgrade CLI functionality! I'm aware of this and will keep it in mind for the parallel PR where we're also updating this CLI (#948) and will fix it there somehow.

For this PR, I think it should be fine.